apps/test/unit/code-studio/pd/form_components_func/FormControllerTest.js

```diff
@@ -20,594 +21,567 @@ let DummyPage3 = () => {
-      );
-      [0, 1, 2].forEach(page => {
-        setPage(page);
-        const buttons = form.findAll('Button');
-        buttons.forEach(button =>
-          expect(button.content()).not.to.eql(saveButtonText)
-        );
-      });
     });
+  });
 
-    it('Shows data was loaded message given application id, and user can close message', () => {
-      form = isolateComponent(
-        <FormController
-          {...defaultProps}
-          applicationId={applicationId}
-          allowPartialSaving={true}
-          validateOnSubmitOnly={true}
-        />
-      );
-      const alert = form.findOne('Alert');
-      expect(alert.content()).to.contain(
-        'We found an application you started! Your saved responses have been loaded.'
-      );
+  it('Shows data was loaded message given application id, and user can close message', () => {
+    form = isolateComponent(
+      <FormController
+        {...defaultProps}
+        applicationId={applicationId}
+        allowPartialSaving={true}
+        validateOnSubmitOnly={true}
+      />
+    );
+    const alert = form.findOne('Alert');
+    expect(alert.content()).to.contain(
+      'We found an application you started! Your saved responses have been loaded.'
+    );
+
+    alert.props.onDismiss();
+    expect(form.exists('Alert')).to.be.false;
+  });
 
-      alert.props.onDismiss();
-      expect(form.exists('Alert')).to.be.false;
-    });
+  it('Shows data was loaded message if status is reopened', () => {
+    form = isolateComponent(
+      <FormController
+        {...defaultProps}
+        applicationId={applicationId}
+        savedStatus={'reopened'}
+        allowPartialSaving={true}
+        validateOnSubmitOnly={true}
+      />
+    );
+    const alert = form.findOne('Alert');
+    expect(alert.content()).to.contain(
+      'Your Regional Partner has requested more information. Please update and resubmit.'
+    );
+
+    alert.props.onDismiss();
+    expect(form.exists('Alert')).to.be.false;
+  });
 
-    it('Shows data was loaded message if status is reopened', () => {
-      form = isolateComponent(
-        <FormController
-          {...defaultProps}
-          applicationId={applicationId}
-          savedStatus={'reopened'}
-          allowPartialSaving={true}
-          validateOnSubmitOnly={true}
-        />
-      );
-      const alert = form.findOne('Alert');
-      expect(alert.content()).to.contain(
-        'Your Regional Partner has requested more information.  Please update and resubmit.'
-      );
+  it('Does not show data was loaded message if there is no application id', () => {
+    form = isolateComponent(
+      <FormController
+        {...defaultProps}
+        allowPartialSaving={true}
+        validateOnSubmitOnly={true}
+      />
+    );
+    expect(form.exists('Alert')).to.be.false;
+  });
 
-      alert.props.onDismiss();
-      expect(form.exists('Alert')).to.be.false;
+  describe('Saving', () => {
+    it('Sends incomplete status on save', () => {
```

Comment:
Refactored to address `// [MEG] TODO: Refactor this to use savedStatus instead of form_data`